servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/NessieInfo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.quarkus.cli;
+
+import com.google.protobuf.ByteString;
+import java.util.stream.Collectors;
+import org.projectnessie.versioned.GetNamedRefsParams;
+import org.projectnessie.versioned.GetNamedRefsParams.RetrieveOptions;
+import org.projectnessie.versioned.ReferenceInfo;
+import org.projectnessie.versioned.persist.adapter.RepoDescription;
+import picocli.CommandLine.Command;
+
+@Command(
+    name = "info",
+    mixinStandardHelpOptions = true,
+    description = "Nessie repository information")
+public class NessieInfo extends BaseCommand {
+
+  @Override
+  public Integer call() throws Exception {
+    warnOnInMemory();
```

Comment:
Info in meaningful for in-memory stores, isn't it? :)

Reply:
> Info is meaningful for in-memory stores, isn't it? :)

For this tool definitely ;)

Reply:
nit: maybe have a `call()` method in `BaseCommand where `warnOnInMemory()` is called, so that this isn't repeated in every command